OsmAnd/src/net/osmand/plus/search/dialogs/QuickSearchDialogFragment.java

```diff
@@ -2103,11 +2103,17 @@ public void onUpdatedIndexesList() {
 		}
 		PluginsHelper.onNewDownloadIndexes(this);
 		updateContent(heading);
+		if (historySearchFragment != null) {
+			historySearchFragment.onUpdatedIndexesList();
+		}
 	}
 
 	@Override
 	public void downloadInProgress() {
 		updateContent(heading);
+		if (historySearchFragment != null) {
+			historySearchFragment.downloadInProgress();
+		}
```

Comment:
Can we check all child fragments similar to mapActivity? in case any other implemented this interface